### PR TITLE
Fix image map on safari

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Fixes `CircleSelection`/`RectangleSelection` performance issues on safari

#### Why?

Because safari is a supported browser and therefore users should not experience any issues when using it.

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/Resources/js/containers/ImageMap/imageRenderer.scss [hunk 1]
```diff
@@ -6,14 +6,15 @@
 .image-renderer-wrapper {
     position: relative;
     overflow: hidden;
-    display: inline-flex;
```
Comment (luca-rath) [inline src/Sulu/Bundle/MediaBundle/Resources/js/containers/ImageMap/imageRenderer.scss:9]: safari somehow doesn't like `inline-flex`

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/ModifiableRectangle.js [hunk 2]
```diff
@@ -67,26 +75,24 @@ class ModifiableRectangle extends React.Component<Props> {
 
     @action handleMouseMove = (event: MouseEvent) => {
         const {onChange} = this.props;
-
-        if (!onChange) {
-            return;
-        }
-
-        const {movementX, movementY} = event;
```
Comment (luca-rath) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/ModifiableRectangle.js:75]: safari's `movementX` behaves different than chrome's `movementX` does. On chrome, `movementX` stops increasing, if the cursor reached the edge of the screen. On safari, `movementX` increases as long as you scroll on the touch pad, even if the cursor already stopped at the edge of the screen.

Comment (luca-rath) [reply src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/ModifiableRectangle.js:75]: ofc same for `movementY`

**Comment by luca-rath** [comment]
@danrot I guess the failing appveyor task isn' related to this pr

**Comment by danrot** [comment]
This also seems to fix the issues I've had with Chrome on linux, which cause the `RectangleSelection` and `CircleSelection` to be moving about twice the speed of the cursor.

**Review by danrot — APPROVED** [review]

**Comment by danrot** [comment]
Going to merge it, since it solves also my chrome problem.

**Pull request merged**